src/main/js/main_html5.js

```diff
@@ -70,10 +70,11 @@ require("../../../html5-common/js/utils/environment.js");
      * @param {string} domId The dom id of the video player instance to create
      * @param {object} controller A reference to the video controller in the Ooyala player
      * @param {object} css The css to apply to the video element
+     * @param {string} playerId An id that represents the player instance
      * @returns {object} A reference to the wrapper for the newly created element
      */
-    this.create = function(parentContainer, domId, controller, css) {
-      if (this.maxSupportedElements > 0 && currentInstances >= this.maxSupportedElements) {
+    this.create = function(parentContainer, domId, controller, css, playerId) {
```

Comment:
You may be thinking of player branding id?  This should be player container in DOM